api/system-status/read

```diff
@@ -54,17 +54,20 @@ function readDisk()
 
 function readRaid()
 {
-    $out = array();
     $ret = array();
-    $out = json_decode(exec('/usr/libexec/nethserver/raid-status'), true);
-    $mds = $out['results'];
-    $ret['status'] = false;
-    for ($i=0; $i<count($mds) && $ret['status'] != 'error'; $i++) {
-      if ($mds[$i]['failed_devs'] == 'none') {
+    $out = array();
+    exec('/usr/libexec/nethserver/raid-status', $out, $exit_code);
```

Comment:
Now I get it: the `$ret['status']` variable is not a boolean, but has 3 different states so you need always to parse the JSON, just what I was trying to avoid ;)

Reply:
I will `change` false to `NULL`, also I prefer it.
I left code as the last commit if it's ok. I think that count number of records to understand if raid is enabled is ok.